packages/engine/src/assets/components/AssetLoader.ts

```diff
@@ -16,7 +16,7 @@ export class AssetLoader extends Component<AssetLoader> {
   /** Class of the asset. Supported Asset classes can be found in {@link assets/enums/AssetClass.AssetClass | AssetClass}. */
   assetClass: AssetClassAlias = null
   /** the object will parse to Colliders when loading location */
-  parseColliders = true
+  dontParseModel = true
```

Comment:
let's not have negatives, can we replace this with `parseModel = false` as default?

Reply:
wait, let's be careful, I may already be confused, the fact is that this is the, default its to parse the model. If we don't know anything about the model yet, we parse it, and if we have already saved everything, then we don't need it anymore.

Reply:
correctly then named, alreadyParsed